CHANGES.md

```diff
@@ -1,32 +1,48 @@
-* Extraction of the operator representing the linear part of PET acquisition model and computation of its norm implemented.
-* Data validity checks return NotImplemented instead of throwing error, opening the door for future implementations of operations on data.
-* Handling of coil images and sensitivities data simplified.
-* STIR projection data now have an implementation of `axpby`, so that duplicate functionality has been removed from SIRF
-* Iterators are now used in `AcquisitionDataInMemory` to improve speed of `as_array`, `fill`, `dot`, `norm`, etc.
-* Bug fix in `get_index_to_physical_point_matrix`
-* if `storage_scheme` is set to `memory`, then `PETAcquisitionData` will now contain `ProjDataInMemory`, instead of `ProjDataFromStream`. As such, if storage scheme is set to memory, it will now be possible to open a PET acquisition data and modify it directly, whereas before a copy would need to be created first.
-* Registration of 2d images with aladin and f3d. 
-* When registering, internally the forward displacement is no loner stored, replaced by the forward deformation. The inverse is no longer stored, and is calculated as needed.
-* When adding a shape to a `sirf.STIR.ImageData`, optionally give the number of times to sample a voxel. This is useful when the shape partially - but not completely - fills a voxel.
-* Renamed `Resample` to `Resampler` and `NiftyResample` to `NiftyResampler`
+* MR acquisition data storage scheme restricted to memory only (a message will be printed but no error thrown)
+* Use CMake variable names from `find_package(Python)` which are available with CMake 3.12+. SIRF CMake files will accept both `Python_EXECUTABLE` or `PYTHON_EXECUTABLE`, for the latter it will send a deprecation warning.
+
+### New features
+* PET
+  - Addition of `sirf.STIR.ScatterEstimation` and `ScatterSimulation` to allow (non-TOF) scatter estimation in PET
+  - GE Signa PET/MR reading of listmode data, sinograms, normalisation and randoms support added.
+  - If STIR is at least version 5 or built from the master branch, [Georg Schramm's parallel (computing) projector](https://github.com/gschramm/parallelproj proj) is now made available from SIRF (use `AcquisitionModelUsingParallelproj`). This uses Joseph interpolation, but importantly can use your GPU (if CUDA was found during building).
+  - Implemented extraction of the operator representing the linear part of PET acquisition model and computation of its norm.
+  - When adding a shape to a `sirf.STIR.ImageData`, optionally give the number of times to sample a voxel. This is useful when the shape partially - but not completely - fills a voxel.
+  - If `storage_scheme` is set to `memory`, `PETAcquisitionData` allows direct modification, whereas before a copy would need to be created first. (Internally, it uses STIR `ProjDataInMemory`, instead of `ProjDataFromStream`).
+* MR
```

Comment:
The existing classes to handle coil image data were mostly deleted and the new ones were inherited from the `ISMRMRDImageData` class. This makes it backwards incompatible only on the C++ level.
On the Python we deleted `CoilImageData` in #646 and then @evgueni-ovtchinnikov restored the class in #786 but it's now names `CoilImagesData` (with an s inbetween), so I think it should not be backwards compatible (although only slightly so).

Reply:
@evgueni-ovtchinnikov can you attempt to rephrase the `CHANGES` accordingly (it's on a branch which you can edit). Move it to the start (backwards incompatible) section.

Reply:
done

Reply:
great. thanks. however (😉 ) as a user I don't know what it means. How does a user need to adapt her code? can you add 2 lines on that as well please?

Reply:
more (hopefully) user-helpful explanation provided